### ISSUE TITLE
About training and the architecure

### ISSUE THREAD

**Issue by chaufanglin** [YUCHEN005/UniVPM#1]
Dear Yuchen,

Nice paper! But I find it hard to follow your repo, I would love to reproduce and see some results, but I don't know if the code is complete. How can we train the model following your pseudo-code in the paper? Same as the other repo: https://github.com/YUCHEN005/MIR-GAN 

Many thanks,
Zhaofeng

**Comment by YUCHEN005** [comment]
Dear Zhaofeng,

Sorry for late reply due to holiday and thank you for attention. You can start from https://github.com/YUCHEN005/UniVPM/tree/master/avhubert/scripts/*.sh, there are two scripts for train and infer respectively. All the code format and styles follow that of AV-Hubert repo (https://github.com/facebookresearch/av_hubert). The thing is the same for another repo you mentioned (see https://github.com/YUCHEN005/MIR-GAN/tree/master/avhubert/scripts).

Best,
Yuchen

**Comment by chaufanglin** [comment]
Thanks for your reply!
